Documentation/Architecture/InputSystem/ControllersPointersAndFocus.md

```diff
@@ -43,10 +43,39 @@ Because a single controller can have multiple pointers (for example, the articul
 For example, as the user’s hand approaches a pressable button, the [`ShellHandRayPointer`](xref:Microsoft.MixedReality.Toolkit.Input.ShellHandRayPointer) should stop showing, and the [`PokePointer`](xref:Microsoft.MixedReality.Toolkit.Input.PokePointer) should be engaged.
 
 This is handled by the [`DefaultPointerMediator`](xref:Microsoft.MixedReality.Toolkit.Input.DefaultPointerMediator),
-which is responsible for determining which pointers based on the state of all pointers. One of the key things this does is [disable far pointers when a near pointer is close to an object](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/DefaultPointerMediator.cs#L127)
+which is responsible for determining which pointers based on the state of all pointers. One of the key things this does is [disable far pointers when a near pointer is close to an object](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/DefaultPointerMediator.cs#L127).
 
 It's possible to provide an alternate implementation of the pointer mediator by changing the [`PointerMediator`](xref:Microsoft.MixedReality.Toolkit.Input.MixedRealityPointerProfile.PointerMediator) property on the pointer profile.
 
+### How to turn pointers on and off in MRTK using PointerBehavior
+Because the pointer mediator runs every frame, it ends up controlling the active / inactive state of all pointers. Therefore, if you set a pointer's IsInteractionEnabled property in code, it will get overwritten by the pointer mediator every frame. Instead, you can specify the [`PointerBehavior`](xref:Microsoft.MixedReality.Toolkit.Inputf.PointerBehavior) to control whether pointers should be on or off yourself. Note that this will only work if you are using the default [`FocusProvider`](xref:Microsoft.MixedReality.Toolkit.Input.FocusProvider) and [`DefaultPointerMediator`](xref:Microsoft.MixedReality.Toolkit.Input.DefaultPointerMediator) in MRTK. 
```

Comment:
Should IsInteractionEnabled be protected/private set then?

Reply:
nit: Remove yourself, you etc

Reply:
That would be a breaking change. Probably for the better since people really , but could break people. @davidkline-ms @wiwei woudl this be an acceptable change, to remove IsInteractionEnabled from being settable publicly?

Reply:
I think it would need to be internal - as a building block (i.e. as a capability that the pointer provides) having IsInteractionEnabled makes sense to be public so that higher level building blocks (of which other MRTK things are one such example) can set/toggle them. 

The value here seems okay I can think of a number of other cases like this, and I don't think we should just do this in a one-off basis.

Reply:
So I would vote no here.

Reply:
Resolving this, I won't be changing it.